variables-cluster.tf

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2017, 2023 Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+variable "create_cluster" {
+  default     = true
+  description = "Whether to create the OKE cluster and dependent resources."
+  type        = bool
+}
+
+variable "cluster_name" {
+  default     = "oke"
+  description = "The name of oke cluster."
+  type        = string
+}
+
+variable "control_plane_type" {
```

Comment:
Should we rename this variable so it has the same terminology as in the console/docs?

Reply:
As with `control_plane_nsg_ids`, this would end up being e.g. `endpoint_config_is_public_ip_enabled`. Aside from being kind of long, it would be a departure from the term used elsewhere that is slightly broader in the cases where we're also referring to the OKE control plane and not just k8s endpoint. Open to considering this more for `5.x`, but I'd propose doing so in a separate change from the file organization here and including all references.